docs/content/en/docs/osmgmt/overview.md

```diff
@@ -0,0 +1,35 @@
+
+|| vSphere | Bare metal | Snow | CloudStack | Nutanix |
+| --- | :---: | :---: | :---: | :---: | :---: |
+| Bottlerocket | &#10004; | &#10004; | &mdash; | &mdash; | &mdash; |
+| Ubuntu | &#10004; | &#10004; | &#10004; | &mdash; | &#10004; |
+| RHEL | &#10004; | &#10004; | &mdash; | &#10004; | &mdash; |
+
+
+With the vSphere, bare metal, Snow, CloudStack and Nutanix deployment options, EKS Anywhere provisions the operating system when new machines are deployed during cluster creation, upgrade, and scaling operations. You can configure the operating system to use through the EKS Anywhere cluster spec, which varies by deployment option. See the deployment option sections below for an overview of how the operating system configuration works per deployment option.
+
+## vSphere
+To configure the operating system to use for EKS Anywhere clusters on vSphere, use the `VSphereMachingConfig` template field. The template name corresponds to the template you imported into your vSphere environment. See the Customize OVAs and Import OVAs documentation pages for more information. Changing the template after cluster creation will result in the deployment of new machines.
+
+## Bare metal
+To configure the operating system to use for EKS Anywhere clusters on bare metal, use the `TinkerbellDatacenterConfig` `osImageURL` field. This field can be used to stream the operating system from a custom location and is required to use Ubuntu or RHEL. You cannot change the `osImageURL` after creating your cluster. To upgrade the operating system, you must replace the image at the existing `osImageURL` location with a new image. Operating system changes are only deployed when an action that triggers a deployment of new machines is triggered, which includes Kubernetes version upgrades only at this time.
+
+## Snow
+To configure the operating to use for EKS Anywhere clusters on Snow, use the `SnowMachineConfig` `osFamily` field. At this time, only Ubuntu is supported for use with EKS Anywhere clusters on Snow. You can customize the instance image with the `SnowMachineConfig` `amiID` field and the instance type with the `SnowMachineConfig` instanceType field. Changes to these fields after cluster creation will result in the deployment of new machines.
+
+## CloudStack
+To configure the operating system to use for EKS Anywhere clusters on CloudStack, use the `CloudStackMachineConfig` template name field. At this time, only RHEL is supported for use with EKS Anywhere clusters on CloudStack. Changing the template name field after cluster creation will result in the deployment of new machines.
+
+## Nutanix
+To configure the operating system to use for EKS Anywhere clusters on Nutanix, use the `NutanixMachineConfig` image name fieldor the image uuid field. At this time, only Ubuntu is supported for use with EKS Anywhere clusters on Nutanix. Changing the image name or uuid field after cluster creation will result in the deployment of new machines.
```

Comment:
@chrisnegus It can be helpful when referencing YAML fields to be explicit about where that field is. For example, the above suggestion uses `spec.image.name` indicating its in the YAML at:

```
spec:
  image:
    name:
```

`name` is a good example because its used multiple times in the same Kubernetes object. Its less relevant when there's only a single field named that way, but it future proofs the docs in-case we add a similar field. Does this work for you? Any other suggestions on referencing fields in YAML explicitly?